dorpsgek_runner/helpers/subprocess.py

```diff
@@ -0,0 +1,26 @@
+import asyncio
+import shlex
+
+
+class CommandError(Exception):
+    """Thrown if the exit code of the command was non-zero."""
+
+
+async def run_command(command, cwd=None):
+    process = await asyncio.create_subprocess_exec(
+        *shlex.split(command),
+        stdout=asyncio.subprocess.PIPE,
+        stderr=asyncio.subprocess.PIPE,
+        cwd=cwd,
+    )
+
+    return_code = await process.wait()
+    if return_code != 0:
+        stderr = []
+        while True:
```

Comment:
Why is this done in an (await) loop? The process has already finished, has it not?